bpf/bpf_lxc.c

```diff
@@ -88,7 +88,7 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 
 	l4_off = l3_off + hdrlen;
 
-#ifndef ENABLE_HOST_SERVICES_FULL
+#if !defined(ENABLE_HOST_SERVICES_FULL) || defined(SKIP_SK_LB_IN_POD_NS)
```

Comment:
nit: We could slightly adapt the logic for not emitting `ENABLE_HOST_SERVICES_FULL` under pkg/datapath/linux/config/config.go.

Reply:
Done.